### DISCUSSION TITLE
Suggestion a VSCode plugin to update front-matter

### DISCUSSION THREAD

**Discussion by mobeicanyue** [hexojs/hexo#5435] [Ideas]
Hey, thanks for developing such a useful blogging tool. I'd like to suggest a feature enhancement for Hexo.

As we're aware, it can be challenging to determine the update time of Markdown files on GitHub Action. 
Therefore, I propose utilizing a front matter to store this information. So we could maintain a VSCode plugin for the task. Whenever we make modifications to an article, the update time would be automatically updated.

This idea was inspired by a plugin I've recently been using, the 'Sitemap Generator'.

**Comment by stevenjoezhang** [comment]
See `updated_option` configuration in https://hexo.io/docs/configuration#Date-x2F-Time-format

**Comment by uiolee** [comment]
>I propose utilizing a front matter to store this information

hexo already has this feat. You can use [`updated`](https://hexo.io/docs/variables#Page-Variables) to store updated time

**Reply by mobeicanyue** [reply]
But when I update markdown, the front formatter `updated` will not be updated. It's the question.
I hope it can update automaticlly, so that github action could manifest correct time.
So an autowritre vscode plugin looks great, like [Sitemap Generator](https://github.com/nils-soderman/vscode-sitemap-generator)
